airplanedb.py

```diff
@@ -178,3 +177,101 @@ def reset_db(self):
         print(('Created new {0} table in {1}').format('SCHEDULE',self.db))
 
         cursor.close()
+
+#==============================================================================
+#   function: update_customer
+#   description: update fields in customer
+#==============================================================================
+    def update_customer(self, custID, field, new_value):
+        db = MySQLdb.connect(host=self.host, 
+                             user=self.user, 
+                             passwd=self.pw, 
+                             db=self.db)
+
+        update_customer_query = """UPDATE CUSTOMER
+                                   SET %s = '%s'
+                                   WHERE C_ID = %s""" % (field, new_value, custID)
+                                   
+        cursor = db.cursor()
+        try:
+            cursor.execute(update_customer_query)
+            db.commit()
+            print("Update Customer Success")
+        except:
+            print("Update Customer Failed")
+            db.rollback()
+        
+        db.close()
+
+#==============================================================================
+#   function: add_baggage
+#   description: adds an instance of baggage to BAGGAGE table
+#==============================================================================
+    def add_baggage(self, bag_ID, cust_ID, bag_weight):
+        db = MySQLdb.connect(host=self.host, 
+                             user=self.user, 
+                             passwd=self.pw, 
+                             db=self.db)
+
+        add_baggage_query = """INSERT INTO BAGGAGE(B_ID,C_ID,B_WEIGHT)
+                                VALUES('%s','%s','%s')""" % (bag_ID, cust_ID, bag_weight)
+
+        cursor = db.cursor()
+        try:
+            cursor.execute(add_baggage_query)
+            db.commit()
+            print("Add Baggage Success")
+        except:
+            print("Add Baggage Failed")
+            db.rollback()
+        
+        db.close()
+
+#==============================================================================
+#   function: add_customer
+#   description: adds an instance of customer to CUSTOMER table
+#==============================================================================
+    def add_customer(self, cust_name, cust_age, cust_email, cust_phone):
+         db = MySQLdb.connect(host=self.host, 
+                             user=self.user, 
+                             passwd=self.pw, 
+                             db=self.db)
+
+         add_customer_query = """INSERT INTO CUSTOMER(C_NAME, C_AGE, C_EMAIL, C_PHONE)
+                                 VALUES('%s','%s','%s','%s')""" % (cust_name, cust_age,
+                                 cust_email, cust_phone)
+
+         cursor = db.cursor()
+         try:
+             cursor.execute(add_customer_query)
+             db.commit()
+             print("Add Customer Success")
```

Comment:
should return the customer ID here so as to give user a reference login number that they can use in the future